examples/example-gauth/src/main/java/io/grpc/examples/googleAuth/GoogleAuthClient.java

```diff
@@ -100,33 +104,29 @@ public void getTopics(String projectID) {
   }
 
   /**
-   * The app requires 2 arguments as described in
-   * @see <a href="../../../../../../GOOGLE_AUTH_EXAMPLE.md">Google Auth Example README</a>
+   * The app requires 1 argument as described in
+   * @see <a href="../../../../../../README.md">Google Auth Example README</a>
    *
-   * arg0 = location of the JSON file for the service account you created in the GCP console
-   * arg1 = project name in the form "projects/balmy-cirrus-225307" where "balmy-cirrus-225307" is
+   * arg0 = project name in the form "projects/balmy-cirrus-225307" where "balmy-cirrus-225307" is
    *        the project ID for the project you created.
```

Comment:
We need to comment where to put the JSON file and what environment variable GOOGLE_APPLICATION_CREDENTIALS to set if they want it somewhere else, and add that the JSON file is unnecessary on Google platforms.

Reply:
Fixed